vital/types/mesh.cxx

```diff
@@ -41,15 +41,16 @@ ::operator==( mesh_vertex_array_base const& other ) const
 
   const std::vector< vector_3d > these_normals = normals();
   const std::vector< vector_3d > other_normals = other.normals();
+  const double limit = 0.000001;
```

Comment:
I'm not a big fan of `operator==` being defined as an approximate equality test and having hard coded thresholds like this.  I think it would be better to redefine these functions as
```c++
bool approx_equal(mesh_vertex_array_base const& array1,
                  mesh_vertex_array_base const& array2,
                  double epsilon = 1e-6)
```
and a similar pattern for the other `operator==`.